File: SortTuplesBySecondElement.cs
// Sort Tuples By Second Element
// - Given a list of tuples, return the list sorted by the second element of each tuple in ascending order.
// - Time: O(n log n), Space: O(n)

using System;
using System.Linq;

public class Solution
{
    public (int, int)[] SortTuplesBySecondElement((int, int)[] tuples)
    {
        return tuples.OrderBy(t => t.Item2).ToArray();
    }
}

File: SortTuplesBySecondElement.py
# Sort Tuples By Second Element
# - Given a list of tuples `pairs`, sort them based on the second element of each tuple.
# - Constraints:
#   - 1 <= len(pairs) <= 10^4

from typing import List, Tuple

class Solution:
    def sortBySecond(self, pairs: List[Tuple[int,int]]) -> List[Tuple[int,int]]:
        return sorted(pairs, key=lambda x: x[1])
